include/ews/ews.hpp

```diff
@@ -10888,6 +10888,39 @@ class attachment final
         return node ? std::string(node->value(), node->value_size()) : "";
     }
 
+    //! \brief Returns the attachment's mime content
+    //!
+    //! If this is a <tt>\<ItemAttachment></tt> and the mime content was
+    //! requested, returns the Base64-encoded mime content of the attachment. If
+    //! this is an \<FileAttachment> or the mime content was not requested,
+    //! returns an empty string.
+    std::string get_mime_content() const
+    {
+        using rapidxml::internal::compare;
+
+        const auto attachment_node = xml_.root();
+        if (!attachment_node)
+        {
+            return nullptr;
```

Comment:
Why not return an empty string in this case? Makes it more difficult to process the function's result, as the caller has first to check if it is a `nullptr` or a (empty) string.

Reply:
How does it even compile? 😯
Ofc it should return an empty string.
I copied from the get_node(...) private function and forget to change this.